openshift-ci/Dockerfile.tools

```diff
@@ -28,10 +28,9 @@ RUN mkdir ~/bin && \
     eval "$(gimme $GOVERSION)" && \
     cat $GIMME_ENV >> $HOME/.bashrc && \
     # get required golang tools and OC client
-    go get github.com/onsi/ginkgo/ginkgo@v1.16.5 && \
-    go get github.com/onsi/gomega/...@v1.17.0 && \
-    go get -u golang.org/x/lint/golint && \
-    go install github.com/lack/mcmaker@v0.0.5 && \
+    go install github.com/onsi/ginkgo/v2/ginkgo@latest && \
```

Comment:
is this wanted? Here we are also bumping ginkgo to v2

Reply:
that's fine, but maybe it belongs to a separate commit, because the commit says "bump go"

Reply:
agree! :)